test/integration/test_notebooks_in_dss_docker_image.py

```diff
@@ -0,0 +1,73 @@
+import io
+import os
+from inspect import cleandoc
+from pathlib import Path
+
+import pytest
+
+from test.integration.docker.exec_run import exec_command
+from test.integration.docker.image import image
+from test.integration.docker.in_memory_build_context import InMemoryBuildContext
+from test.integration.docker.container import container
+
+TEST_RESOURCE_PATH = Path(__file__).parent.parent / "notebooks"
+
+
+@pytest.fixture(scope="session")
+def notebook_test_dockerfile_content(dss_docker_image) -> str:
+    yield cleandoc(
+        f"""
+        FROM {dss_docker_image.image_name} 
+        COPY notebooks/* /tmp/notebooks/
+        RUN cp /tmp/notebooks/* "$NOTEBOOK_FOLDER_INITIAL"
```

Comment:
Is it possible to copy to the correct place in one command?
If not: use `mv` or `cp && rm` to clean up the temp files.

Reply:
It is not possible to use the environment variable in the copy directive. Removing the tmp directory doesn't really have an effect on the size of the container, it only cleans up the final file system. One thing, that works is copying the files first to `/dev/shm`, because this is mounted as tmpfs, however its size might be very limited, for that reason, I don't see this as a good solution. Overall, it doesn't matter for the purpose of testing. I will remove the tmp directory anyway, but to clear the file system.

Reply:
done